### ISSUE TITLE
Make changing location on client toggleable

### ISSUE THREAD

**Issue by cschmatzler** [AHAAAAAAA/PokemonGo-Map#1517]
Currently, everybody can change the search location from the website. 
I am hosting the map for my city for a Facebook group and, as you can think, this feature isn't exactly helpful if random people change the search location to whatever location, effectively disabling the service for others.
Are there any plans to make this feature toggleable from the server side, maybe with another argument?

**Comment by viruzofficial** [comment]
How can you change Location? When i enter a new Location and press enter, nothing happens.

**Comment by docrockulus** [comment]
I started up an instance without the scanning thread (-m) and set that as my web interface. I think this will resolve the issue as users won't be stealing a scanning thread. 

I'll update with my results.

**Comment by docrockulus** [comment]
Yeah, looks like it worked. None of the scrubs in my city have been able to steal it yet.

**Comment by CtrlAltDefeat94** [comment]
Yes, please implement this

**Comment by docrockulus** [comment]
just do it when you start your instances:

`python runserver.py -a google -u uuu -p ppp -l "lat long" -st 1 -H 0.0.0.0 -P 80 -m`
then start the rest of your instances like you normally would
`python runserver.py -a google -u uuu -p ppp -l "lat long" -st 10 -H 0.0.0.0 -P 5000 -t 3`
`rest of instances...`

**Comment by timendum** [comment]
This is implemented already, can we close this issue?
